Makefile

```diff
@@ -33,6 +33,7 @@ vet:
 
 build:
 	@echo ">> building binaries"
+	@./build/assets.sh
```

Comment:
How long does this take? If it's slow, we might want to make it a separate rule, since UI changes are relatively infrequent.